repo2docker/buildpacks/conda/environment.py-3.5.frozen.yml

```diff
@@ -1,79 +1,9 @@
 # AUTO GENERATED FROM environment.py-3.5.yml, DO NOT MANUALLY MODIFY
-# Frozen on 2018-07-16 23:05:45 UTC
+# Frozen on 2018-09-07 15:52:23 UTC
 name: r2d
 channels:
   - conda-forge
   - defaults
   - conda-forge/label/broken
-dependencies:
```

Comment:
No more dependencies at all for PYthon 3.5?

Reply:
@betatim yeah, was a bug in freeze.py. Fixed.